asistente_ladm_col/utils/quality.py

```diff
@@ -471,10 +474,12 @@ def check_too_long_segments(self, db):
     def check_missing_boundary_points_in_boundaries(self, db):
         res_layers = self.qgis_utils.get_layers(db, {
             BOUNDARY_POINT_TABLE: {'name': BOUNDARY_POINT_TABLE, 'geometry': None},
+            POINT_BOUNDARY_FACE_STRING_TABLE: {'name': POINT_BOUNDARY_FACE_STRING_TABLE, 'geometry': None},
             BOUNDARY_TABLE: {'name': BOUNDARY_TABLE, 'geometry': None}}, load=True)
 
         boundary_point_layer = res_layers[BOUNDARY_POINT_TABLE]
         boundary_layer = res_layers[BOUNDARY_TABLE]
+        pointCcl_table = res_layers[POINT_BOUNDARY_FACE_STRING_TABLE]
```

Comment:
Please stick to lower-case letters in variable names.

Reply:
it was ajusted